README.md

```diff
@@ -33,6 +34,15 @@ If you encounter the following error:
 The following command should fix your issue
 `createdb -h localhost`
 
+To run tests, you will need this:
+`createdb -h localhost lacuna`
```

Comment:
Seeing as the contributions in this repo are intended to be merged back into OMF at some point, we probably don't want to specify anything about `lacuna` in here. Perhaps `mds`?

Reply:
Gotcha.  Will make this more generic.